{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "42fe8e6b",
   "metadata": {},
   "source": [
    "### QIIME2 HANDS-ON TUTORIAL"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a5acfc70",
   "metadata": {},
   "source": [
    "You’ll use QIIME 2 to perform an analysis of human microbiome samples from two individuals at four body sites at five timepoints, the first of which immediately followed antibiotic usage. \n",
    "A study based on these samples was originally published in Caporaso et al. (2011). \n",
    "\n",
    "The data used in this tutorial were sequenced on an Illumina HiSeq using the Earth Microbiome Project hypervariable region 4 (V4) 16S rRNA sequencing protocol."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "954d2e64",
   "metadata": {},
   "source": [
    "### 2 Importing fastq files to create Artifacts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "2126a322",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mImported /mnt/Mammal_dataset/raw_data as CasavaOneEightSingleLanePerSampleDirFmt to demux.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime tools import \\\n",
    "  --type 'SampleData[PairedEndSequencesWithQuality]' \\\n",
    "  --input-path /mnt/Mammal_dataset/raw_data \\\n",
    "  --input-format CasavaOneEightSingleLanePerSampleDirFmt \\\n",
    "  --output-path demux.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c601102",
   "metadata": {},
   "source": [
    "### 3 Trime sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "52b4d334",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY001_F12_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY001_F12_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY001_F12_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY001_F12_R02_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY001_F12_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY001_F12_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY001_F12_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY001_F12_R02_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        27,211 reads @  28.0 µs/read;   2.14 M reads/minute\n",
      "Finished in 0.764 s (28.088 µs/read; 2.14 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,211\n",
      "  Read 1 with adapter:                  27,140 (99.7%)\n",
      "  Read 2 with adapter:                  27,138 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):        27,070 (99.5%)\n",
      "\n",
      "Total basepairs processed:    14,844,708 bp\n",
      "  Read 1:     7,875,555 bp\n",
      "  Read 2:     6,969,153 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,095,874 bp (81.5%)\n",
      "  Read 1:     6,526,168 bp\n",
      "  Read 2:     5,569,706 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27140 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t0 1\n",
      "41\t6\t0.0\t2\t6\n",
      "42\t1\t0.0\t2\t1\n",
      "43\t4\t0.0\t2\t2 2\n",
      "44\t6\t0.0\t2\t3 3\n",
      "45\t23\t0.0\t2\t11 9 3\n",
      "46\t29\t0.0\t2\t28 1\n",
      "47\t169\t0.0\t2\t127 21 21\n",
      "48\t1417\t0.0\t2\t913 485 19\n",
      "49\t25391\t0.0\t2\t24415 952 24\n",
      "50\t92\t0.0\t2\t66 25 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 27138 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "42\t1\t0.0\t3\t1\n",
      "44\t3\t0.0\t3\t3\n",
      "45\t2\t0.0\t3\t2\n",
      "46\t4\t0.0\t3\t4\n",
      "47\t8\t0.0\t3\t5 1 2\n",
      "48\t24\t0.0\t3\t14 4 0 6\n",
      "49\t127\t0.0\t3\t91 23 12 1\n",
      "50\t1333\t0.0\t3\t831 476 24 2\n",
      "51\t25379\t0.0\t3\t22961 2268 145 5\n",
      "52\t254\t0.0\t3\t204 41 9\n",
      "53\t3\t0.0\t3\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY005_F08_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY005_F08_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY005_F08_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY005_F08_R02_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY005_F08_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY005_F08_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY005_F08_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY005_F08_R02_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        19,021 reads @  27.0 µs/read;   2.22 M reads/minute\n",
      "Finished in 0.516 s (27.111 µs/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,021\n",
      "  Read 1 with adapter:                  18,968 (99.7%)\n",
      "  Read 2 with adapter:                  18,975 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        18,927 (99.5%)\n",
      "\n",
      "Total basepairs processed:    10,406,995 bp\n",
      "  Read 1:     5,464,488 bp\n",
      "  Read 2:     4,942,507 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,485,929 bp (81.5%)\n",
      "  Read 1:     4,521,147 bp\n",
      "  Read 2:     3,964,782 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18968 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t2\t0.0\t2\t2\n",
      "41\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t0 0 1\n",
      "43\t3\t0.0\t2\t3\n",
      "44\t5\t0.0\t2\t3 1 1\n",
      "45\t5\t0.0\t2\t3 2\n",
      "46\t17\t0.0\t2\t12 3 2\n",
      "47\t170\t0.0\t2\t137 14 19\n",
      "48\t1251\t0.0\t2\t873 362 16\n",
      "49\t17323\t0.0\t2\t16568 733 22\n",
      "50\t188\t0.0\t2\t161 27\n",
      "51\t2\t0.0\t2\t1 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18975 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t1\t0.0\t3\t1\n",
      "45\t1\t0.0\t3\t1\n",
      "46\t2\t0.0\t3\t1 1\n",
      "47\t6\t0.0\t3\t6\n",
      "48\t16\t0.0\t3\t11 3 0 2\n",
      "49\t79\t0.0\t3\t48 19 12\n",
      "50\t906\t0.0\t3\t542 333 28 3\n",
      "51\t17770\t0.0\t3\t15991 1669 103 7\n",
      "52\t188\t0.0\t3\t154 29 5\n",
      "53\t5\t0.0\t3\t5\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY009_F12_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY009_F12_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY009_F12_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY009_F12_R03_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY009_F12_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY009_F12_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY009_F12_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY009_F12_R03_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        27,141 reads @  28.2 µs/read;   2.13 M reads/minute\n",
      "Finished in 0.767 s (28.260 µs/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             27,141\n",
      "  Read 1 with adapter:                  27,065 (99.7%)\n",
      "  Read 2 with adapter:                  27,057 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   7 (0.0%)\n",
      "Pairs written (passing filters):        26,995 (99.5%)\n",
      "\n",
      "Total basepairs processed:    14,870,224 bp\n",
      "  Read 1:     7,909,337 bp\n",
      "  Read 2:     6,960,887 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,102,932 bp (81.4%)\n",
      "  Read 1:     6,562,771 bp\n",
      "  Read 2:     5,540,161 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 27065 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "38\t1\t0.0\t2\t1\n",
      "39\t3\t0.0\t2\t3\n",
      "40\t2\t0.0\t2\t2\n",
      "41\t4\t0.0\t2\t3 1\n",
      "42\t3\t0.0\t2\t3\n",
      "43\t4\t0.0\t2\t3 1\n",
      "44\t8\t0.0\t2\t8\n",
      "45\t10\t0.0\t2\t7 3\n",
      "46\t37\t0.0\t2\t32 4 1\n",
      "47\t183\t0.0\t2\t138 25 20\n",
      "48\t1409\t0.0\t2\t881 512 16\n",
      "49\t25336\t0.0\t2\t24369 942 25\n",
      "50\t64\t0.0\t2\t41 23\n",
      "51\t1\t0.0\t2\t0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 27057 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t2\t0.0\t3\t2\n",
      "43\t1\t0.0\t3\t1\n",
      "45\t2\t0.0\t3\t2\n",
      "47\t5\t0.0\t3\t4 1\n",
      "48\t5\t0.0\t3\t3 2\n",
      "49\t24\t0.0\t3\t12 4 2 6\n",
      "50\t134\t0.0\t3\t96 21 16 1\n",
      "51\t1379\t0.0\t3\t881 453 43 2\n",
      "52\t25247\t0.0\t3\t22721 2380 134 12\n",
      "53\t254\t0.0\t3\t219 31 4\n",
      "54\t4\t0.0\t3\t2 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY012_F08_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY012_F08_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY012_F08_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY012_F08_R03_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY012_F08_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY012_F08_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY012_F08_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY012_F08_R03_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        20,126 reads @  27.6 µs/read;   2.17 M reads/minute\n",
      "Finished in 0.558 s (27.728 µs/read; 2.16 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             20,126\n",
      "  Read 1 with adapter:                  20,070 (99.7%)\n",
      "  Read 2 with adapter:                  20,062 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        20,011 (99.4%)\n",
      "\n",
      "Total basepairs processed:    10,951,500 bp\n",
      "  Read 1:     5,832,498 bp\n",
      "  Read 2:     5,119,002 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,896,326 bp (81.2%)\n",
      "  Read 1:     4,832,300 bp\n",
      "  Read 2:     4,064,026 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 20070 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t2\t1\n",
      "39\t2\t0.0\t2\t2\n",
      "40\t1\t0.0\t2\t0 1\n",
      "41\t1\t0.0\t2\t1\n",
      "44\t5\t0.0\t2\t3 2\n",
      "45\t12\t0.0\t2\t6 5 1\n",
      "46\t32\t0.0\t2\t25 6 1\n",
      "47\t188\t0.0\t2\t131 29 28\n",
      "48\t1325\t0.0\t2\t907 403 15\n",
      "49\t18307\t0.0\t2\t17580 710 17\n",
      "50\t193\t0.0\t2\t162 31\n",
      "51\t3\t0.0\t2\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 20062 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "44\t1\t0.0\t3\t1\n",
      "45\t1\t0.0\t3\t1\n",
      "47\t1\t0.0\t3\t1\n",
      "48\t3\t0.0\t3\t1 1 1\n",
      "49\t18\t0.0\t3\t7 4 4 3\n",
      "50\t104\t0.0\t3\t75 17 12\n",
      "51\t962\t0.0\t3\t612 317 30 3\n",
      "52\t18753\t0.0\t3\t16963 1689 95 6\n",
      "53\t218\t0.0\t3\t185 31 2\n",
      "54\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY016_F12_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY016_F12_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY016_F12_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY016_F12_R04_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY016_F12_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/BY016_F12_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY016_F12_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/BY016_F12_R04_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        28,708 reads @  26.2 µs/read;   2.29 M reads/minute\n",
      "Finished in 0.754 s (26.250 µs/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,708\n",
      "  Read 1 with adapter:                  28,655 (99.8%)\n",
      "  Read 2 with adapter:                  28,666 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                  10 (0.0%)\n",
      "Pairs written (passing filters):        28,608 (99.7%)\n",
      "\n",
      "Total basepairs processed:    15,529,264 bp\n",
      "  Read 1:     8,347,210 bp\n",
      "  Read 2:     7,182,054 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,693,695 bp (81.7%)\n",
      "  Read 1:     6,927,428 bp\n",
      "  Read 2:     5,766,267 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28655 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t2\t1\n",
      "37\t1\t0.0\t2\t1\n",
      "40\t2\t0.0\t2\t2\n",
      "41\t5\t0.0\t2\t5\n",
      "42\t6\t0.0\t2\t5 1\n",
      "43\t5\t0.0\t2\t4 1\n",
      "44\t10\t0.0\t2\t8 2\n",
      "45\t24\t0.0\t2\t19 5\n",
      "46\t36\t0.0\t2\t34 2\n",
      "47\t194\t0.0\t2\t134 22 38\n",
      "48\t1469\t0.0\t2\t880 563 26\n",
      "49\t26825\t0.0\t2\t25827 977 21\n",
      "50\t76\t0.0\t2\t36 39 1\n",
      "73\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 28666 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t3\t1\n",
      "41\t1\t0.0\t3\t1\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t2\t0.0\t3\t1 1\n",
      "45\t5\t0.0\t3\t5\n",
      "46\t23\t0.0\t3\t13 3 4 3\n",
      "47\t147\t0.0\t3\t98 30 19\n",
      "48\t1437\t0.0\t3\t889 503 42 3\n",
      "49\t26835\t0.0\t3\t24216 2477 135 7\n",
      "50\t212\t0.0\t3\t178 30 4\n",
      "51\t2\t0.0\t3\t1 0 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext001_F01_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext001_F01_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext001_F01_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext001_F01_R01_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext001_F01_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext001_F01_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext001_F01_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext001_F01_R01_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00            73 reads @ 138.8 µs/read;   0.43 M reads/minute\n",
      "Finished in 0.012 s (160.933 µs/read; 0.37 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 73\n",
      "  Read 1 with adapter:                      70 (95.9%)\n",
      "  Read 2 with adapter:                      70 (95.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):            70 (95.9%)\n",
      "\n",
      "Total basepairs processed:        38,880 bp\n",
      "  Read 1:        20,642 bp\n",
      "  Read 2:        18,238 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         30,278 bp (77.9%)\n",
      "  Read 1:        16,280 bp\n",
      "  Read 2:        13,998 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 70 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t6\t0.0\t2\t3 3\n",
      "50\t63\t0.0\t2\t60 3\n",
      "51\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 70 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t4\t0.0\t3\t0 3 1\n",
      "50\t66\t0.0\t3\t55 11\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext002_F01_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext002_F01_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext002_F01_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext002_F01_R02_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext002_F01_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext002_F01_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext002_F01_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext002_F01_R02_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00           100 reads @ 114.6 µs/read;   0.52 M reads/minute\n",
      "Finished in 0.013 s (132.287 µs/read; 0.45 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                100\n",
      "  Read 1 with adapter:                      86 (86.0%)\n",
      "  Read 2 with adapter:                      87 (87.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):            86 (86.0%)\n",
      "\n",
      "Total basepairs processed:        51,560 bp\n",
      "  Read 1:        27,678 bp\n",
      "  Read 2:        23,882 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         36,465 bp (70.7%)\n",
      "  Read 1:        19,786 bp\n",
      "  Read 2:        16,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 86 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "48\t3\t0.0\t2\t2 1\n",
      "49\t5\t0.0\t2\t5\n",
      "50\t78\t0.0\t2\t73 5\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 87 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t1\t0.0\t3\t0 1\n",
      "50\t3\t0.0\t3\t2 1\n",
      "51\t81\t0.0\t3\t69 11 1\n",
      "52\t2\t0.0\t3\t1 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext003_F03_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext003_F03_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext003_F03_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext003_F03_R11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext003_F03_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext003_F03_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext003_F03_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext003_F03_R11_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00            76 reads @ 125.1 µs/read;   0.48 M reads/minute\n",
      "Finished in 0.012 s (154.752 µs/read; 0.39 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                 76\n",
      "  Read 1 with adapter:                      75 (98.7%)\n",
      "  Read 2 with adapter:                      75 (98.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):            75 (98.7%)\n",
      "\n",
      "Total basepairs processed:        38,527 bp\n",
      "  Read 1:        20,331 bp\n",
      "  Read 2:        18,196 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         30,462 bp (79.1%)\n",
      "  Read 1:        16,173 bp\n",
      "  Read 2:        14,289 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 75 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "50\t1\t0.0\t2\t1\n",
      "51\t2\t0.0\t2\t1 1\n",
      "52\t71\t0.0\t2\t68 3\n",
      "53\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 75 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "48\t1\t0.0\t3\t0 1\n",
      "49\t72\t0.0\t3\t60 10 2\n",
      "50\t2\t0.0\t3\t1 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext004_F06_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext004_F06_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext004_F06_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext004_F06_R04_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext004_F06_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext004_F06_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext004_F06_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext004_F06_R04_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00           167 reads @  81.1 µs/read;   0.74 M reads/minute\n",
      "Finished in 0.016 s (92.818 µs/read; 0.65 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                167\n",
      "  Read 1 with adapter:                     164 (98.2%)\n",
      "  Read 2 with adapter:                     164 (98.2%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           164 (98.2%)\n",
      "\n",
      "Total basepairs processed:        87,994 bp\n",
      "  Read 1:        47,720 bp\n",
      "  Read 2:        40,274 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):         71,335 bp (81.1%)\n",
      "  Read 1:        39,288 bp\n",
      "  Read 2:        32,047 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 164 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "47\t1\t0.0\t2\t0 0 1\n",
      "48\t15\t0.0\t2\t10 5\n",
      "49\t145\t0.0\t2\t139 6\n",
      "50\t3\t0.0\t2\t2 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 164 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "48\t4\t0.0\t3\t2 1 1\n",
      "49\t158\t0.0\t3\t142 15 1\n",
      "50\t2\t0.0\t3\t1 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext005_F10_R09_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext005_F10_R09_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext005_F10_R09_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext005_F10_R09_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext005_F10_R09_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/Ext005_F10_R09_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext005_F10_R09_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/Ext005_F10_R09_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00           499 reads @  48.2 µs/read;   1.24 M reads/minute\n",
      "Finished in 0.027 s (53.707 µs/read; 1.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                499\n",
      "  Read 1 with adapter:                     493 (98.8%)\n",
      "  Read 2 with adapter:                     492 (98.6%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           492 (98.6%)\n",
      "\n",
      "Total basepairs processed:       263,706 bp\n",
      "  Read 1:       143,822 bp\n",
      "  Read 2:       119,884 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):        212,255 bp (80.5%)\n",
      "  Read 1:       117,369 bp\n",
      "  Read 2:        94,886 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 493 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t8\t0.0\t2\t4 3 1\n",
      "50\t23\t0.0\t2\t14 9\n",
      "51\t459\t0.0\t2\t442 17\n",
      "52\t3\t0.0\t2\t0 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 492 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t1\t0.0\t3\t1\n",
      "48\t27\t0.0\t3\t15 11 1\n",
      "49\t461\t0.0\t3\t400 54 6 1\n",
      "50\t3\t0.0\t3\t3\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD015_F06_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD015_F06_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD015_F06_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD015_F06_R05_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD015_F06_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD015_F06_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD015_F06_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD015_F06_R05_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        16,934 reads @  27.1 µs/read;   2.22 M reads/minute\n",
      "Finished in 0.460 s (27.174 µs/read; 2.21 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,934\n",
      "  Read 1 with adapter:                  16,891 (99.7%)\n",
      "  Read 2 with adapter:                  16,884 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   3 (0.0%)\n",
      "Pairs written (passing filters):        16,849 (99.5%)\n",
      "\n",
      "Total basepairs processed:     9,202,452 bp\n",
      "  Read 1:     4,929,039 bp\n",
      "  Read 2:     4,273,413 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      7,510,938 bp (81.6%)\n",
      "  Read 1:     4,090,307 bp\n",
      "  Read 2:     3,420,631 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16891 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t2\t1\n",
      "44\t2\t0.0\t2\t2\n",
      "45\t4\t0.0\t2\t3 1\n",
      "46\t18\t0.0\t2\t13 4 1\n",
      "47\t154\t0.0\t2\t113 24 17\n",
      "48\t1147\t0.0\t2\t869 266 12\n",
      "49\t15419\t0.0\t2\t14770 640 9\n",
      "50\t142\t0.0\t2\t118 24\n",
      "51\t4\t0.0\t2\t3 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16884 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t3\t1\n",
      "41\t1\t0.0\t3\t1\n",
      "42\t1\t0.0\t3\t1\n",
      "44\t1\t0.0\t3\t1\n",
      "45\t1\t0.0\t3\t1\n",
      "46\t5\t0.0\t3\t4 0 0 1\n",
      "47\t13\t0.0\t3\t8 1 1 3\n",
      "48\t56\t0.0\t3\t34 11 11\n",
      "49\t738\t0.0\t3\t418 294 25 1\n",
      "50\t15891\t0.0\t3\t14008 1743 127 13\n",
      "51\t173\t0.0\t3\t137 33 3\n",
      "52\t3\t0.0\t3\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD016_F10_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD016_F10_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD016_F10_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD016_F10_R11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD016_F10_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD016_F10_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD016_F10_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD016_F10_R11_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        19,177 reads @  26.0 µs/read;   2.31 M reads/minute\n",
      "Finished in 0.501 s (26.129 µs/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,177\n",
      "  Read 1 with adapter:                  19,130 (99.8%)\n",
      "  Read 2 with adapter:                  19,130 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):        19,080 (99.5%)\n",
      "\n",
      "Total basepairs processed:    10,462,511 bp\n",
      "  Read 1:     5,592,674 bp\n",
      "  Read 2:     4,869,837 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,523,813 bp (81.5%)\n",
      "  Read 1:     4,602,047 bp\n",
      "  Read 2:     3,921,766 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19130 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t2\t1\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "41\t2\t0.0\t2\t2\n",
      "42\t1\t0.0\t2\t1\n",
      "43\t5\t0.0\t2\t4 1\n",
      "44\t2\t0.0\t2\t2\n",
      "45\t6\t0.0\t2\t3 3\n",
      "46\t5\t0.0\t2\t3 1 1\n",
      "47\t12\t0.0\t2\t10 1 1\n",
      "48\t37\t0.0\t2\t34 2 1\n",
      "49\t175\t0.0\t2\t129 20 26\n",
      "50\t1027\t0.0\t2\t703 318 6\n",
      "51\t17808\t0.0\t2\t17320 485 3\n",
      "52\t47\t0.0\t2\t31 15 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19130 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t3\t0 1\n",
      "41\t1\t0.0\t3\t1\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t4\t0.0\t3\t4\n",
      "45\t4\t0.0\t3\t2 0 1 1\n",
      "46\t13\t0.0\t3\t6 2 2 3\n",
      "47\t85\t0.0\t3\t52 19 14\n",
      "48\t935\t0.0\t3\t565 345 24 1\n",
      "49\t17878\t0.0\t3\t16180 1583 111 4\n",
      "50\t203\t0.0\t3\t174 26 3\n",
      "51\t4\t0.0\t3\t2 2\n",
      "69\t1\t0.0\t3\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD017_F10_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD017_F10_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD017_F10_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD017_F10_R12_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD017_F10_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD017_F10_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD017_F10_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD017_F10_R12_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        13,226 reads @  26.9 µs/read;   2.23 M reads/minute\n",
      "Finished in 0.358 s (27.045 µs/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             13,226\n",
      "  Read 1 with adapter:                  13,201 (99.8%)\n",
      "  Read 2 with adapter:                  13,187 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):        13,164 (99.5%)\n",
      "\n",
      "Total basepairs processed:     7,260,263 bp\n",
      "  Read 1:     3,857,104 bp\n",
      "  Read 2:     3,403,159 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      5,900,363 bp (81.3%)\n",
      "  Read 1:     3,175,827 bp\n",
      "  Read 2:     2,724,536 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 13201 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t4\t0.0\t2\t3 1\n",
      "41\t1\t0.0\t2\t0 1\n",
      "42\t3\t0.0\t2\t2 0 1\n",
      "43\t2\t0.0\t2\t2\n",
      "44\t2\t0.0\t2\t2\n",
      "45\t5\t0.0\t2\t5\n",
      "46\t8\t0.0\t2\t8\n",
      "47\t8\t0.0\t2\t8\n",
      "48\t19\t0.0\t2\t15 4\n",
      "49\t110\t0.0\t2\t91 7 12\n",
      "50\t617\t0.0\t2\t391 221 5\n",
      "51\t12387\t0.0\t2\t11966 409 12\n",
      "52\t35\t0.0\t2\t18 15 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 13187 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "42\t3\t0.0\t3\t3\n",
      "45\t3\t0.0\t3\t3\n",
      "47\t2\t0.0\t3\t1 0 1\n",
      "48\t10\t0.0\t3\t8 2\n",
      "49\t50\t0.0\t3\t27 9 13 1\n",
      "50\t577\t0.0\t3\t342 214 18 3\n",
      "51\t12396\t0.0\t3\t11184 1139 69 4\n",
      "52\t142\t0.0\t3\t119 23\n",
      "53\t4\t0.0\t3\t3 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD018_F11_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD018_F11_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD018_F11_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD018_F11_R01_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD018_F11_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD018_F11_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD018_F11_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD018_F11_R01_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        23,352 reads @  28.7 µs/read;   2.09 M reads/minute\n",
      "Finished in 0.672 s (28.764 µs/read; 2.09 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,352\n",
      "  Read 1 with adapter:                  23,352 (100.0%)\n",
      "  Read 2 with adapter:                  23,279 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):        23,272 (99.7%)\n",
      "\n",
      "Total basepairs processed:    12,857,797 bp\n",
      "  Read 1:     6,834,227 bp\n",
      "  Read 2:     6,023,570 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     10,468,129 bp (81.4%)\n",
      "  Read 1:     5,613,450 bp\n",
      "  Read 2:     4,854,679 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23352 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t79\t0.4\t1\t0 79\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t1\n",
      "41\t4\t0.0\t2\t4\n",
      "42\t5\t0.0\t2\t5\n",
      "43\t7\t0.0\t2\t2 5\n",
      "44\t3\t0.0\t2\t2 1\n",
      "45\t9\t0.0\t2\t7 2\n",
      "46\t7\t0.0\t2\t7\n",
      "47\t16\t0.0\t2\t16\n",
      "48\t26\t0.0\t2\t23 2 1\n",
      "49\t55\t0.0\t2\t45 6 4\n",
      "50\t204\t0.0\t2\t160 29 15\n",
      "51\t1346\t0.0\t2\t940 397 9\n",
      "52\t21527\t0.0\t2\t20979 534 14\n",
      "53\t59\t0.0\t2\t35 23 1\n",
      "63\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 23279 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "41\t1\t0.0\t3\t0 1\n",
      "42\t1\t0.0\t3\t1\n",
      "43\t3\t0.0\t3\t1 2\n",
      "45\t6\t0.0\t3\t5 1\n",
      "46\t4\t0.0\t3\t4\n",
      "47\t9\t0.0\t3\t4 3 1 1\n",
      "48\t99\t0.0\t3\t58 18 20 3\n",
      "49\t899\t0.0\t3\t503 361 33 2\n",
      "50\t22040\t0.0\t3\t19774 2136 128 2\n",
      "51\t217\t0.0\t3\t169 47 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD019_F06_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD019_F06_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD019_F06_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD019_F06_R06_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD019_F06_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD019_F06_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD019_F06_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD019_F06_R06_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        16,231 reads @  28.3 µs/read;   2.12 M reads/minute\n",
      "Finished in 0.460 s (28.357 µs/read; 2.12 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,231\n",
      "  Read 1 with adapter:                  16,191 (99.8%)\n",
      "  Read 2 with adapter:                  16,189 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        16,148 (99.5%)\n",
      "\n",
      "Total basepairs processed:     8,885,713 bp\n",
      "  Read 1:     4,729,999 bp\n",
      "  Read 2:     4,155,714 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      7,278,651 bp (81.9%)\n",
      "  Read 1:     3,924,205 bp\n",
      "  Read 2:     3,354,446 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16191 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "44\t3\t0.0\t2\t2 0 1\n",
      "45\t5\t0.0\t2\t2 3\n",
      "46\t22\t0.0\t2\t18 4\n",
      "47\t113\t0.0\t2\t79 24 10\n",
      "48\t1141\t0.0\t2\t838 296 7\n",
      "49\t14769\t0.0\t2\t14255 507 7\n",
      "50\t135\t0.0\t2\t112 22 1\n",
      "51\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16189 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t2\t0.0\t3\t2\n",
      "42\t1\t0.0\t3\t1\n",
      "43\t3\t0.0\t3\t3\n",
      "44\t3\t0.0\t3\t1 2\n",
      "45\t5\t0.0\t3\t4 1\n",
      "46\t7\t0.0\t3\t5 2\n",
      "47\t83\t0.0\t3\t46 22 13 2\n",
      "48\t706\t0.0\t3\t368 292 42 4\n",
      "49\t15240\t0.0\t3\t12185 2788 251 16\n",
      "50\t139\t0.0\t3\t98 36 4 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD028_F06_R07_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD028_F06_R07_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD028_F06_R07_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD028_F06_R07_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD028_F06_R07_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GD028_F06_R07_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD028_F06_R07_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GD028_F06_R07_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:01        70,108 reads @  25.7 µs/read;   2.33 M reads/minute\n",
      "Finished in 1.806 s (25.759 µs/read; 2.33 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             70,108\n",
      "  Read 1 with adapter:                  69,708 (99.4%)\n",
      "  Read 2 with adapter:                  69,684 (99.4%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                  23 (0.0%)\n",
      "Pairs written (passing filters):        69,525 (99.2%)\n",
      "\n",
      "Total basepairs processed:    38,134,213 bp\n",
      "  Read 1:    20,364,463 bp\n",
      "  Read 2:    17,769,750 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     30,957,858 bp (81.2%)\n",
      "  Read 1:    16,861,752 bp\n",
      "  Read 2:    14,096,106 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 69708 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t2\t1\n",
      "41\t2\t0.0\t2\t2\n",
      "42\t3\t0.0\t2\t3\n",
      "43\t1\t0.0\t2\t1\n",
      "44\t5\t0.0\t2\t5\n",
      "45\t9\t0.0\t2\t6 2 1\n",
      "46\t76\t0.0\t2\t63 8 5\n",
      "47\t489\t0.0\t2\t364 73 52\n",
      "48\t4531\t0.0\t2\t3239 1241 51\n",
      "49\t63966\t0.0\t2\t61088 2804 74\n",
      "50\t622\t0.0\t2\t533 87 2\n",
      "51\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 69684 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t3\t1\n",
      "38\t4\t0.0\t3\t4\n",
      "40\t2\t0.0\t3\t2\n",
      "41\t2\t0.0\t3\t2\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t3\t0.0\t3\t2 1\n",
      "45\t2\t0.0\t3\t2\n",
      "46\t3\t0.0\t3\t3\n",
      "47\t8\t0.0\t3\t6 1 0 1\n",
      "48\t14\t0.0\t3\t10 3 1\n",
      "49\t46\t0.0\t3\t24 6 6 10\n",
      "50\t286\t0.0\t3\t171 51 61 3\n",
      "51\t3239\t0.0\t3\t1960 1148 126 5\n",
      "52\t65349\t0.0\t3\t58109 6735 457 48\n",
      "53\t659\t0.0\t3\t536 112 11\n",
      "54\t59\t0.0\t3\t54 4 1\n",
      "55\t3\t0.0\t3\t2 1\n",
      "56\t3\t0.0\t3\t3\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW021_F11_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW021_F11_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW021_F11_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW021_F11_R02_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW021_F11_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW021_F11_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW021_F11_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW021_F11_R02_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        23,932 reads @  29.2 µs/read;   2.06 M reads/minute\n",
      "Finished in 0.701 s (29.273 µs/read; 2.05 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             23,932\n",
      "  Read 1 with adapter:                  23,932 (100.0%)\n",
      "  Read 2 with adapter:                  23,874 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   6 (0.0%)\n",
      "Pairs written (passing filters):        23,869 (99.7%)\n",
      "\n",
      "Total basepairs processed:    13,172,461 bp\n",
      "  Read 1:     6,946,846 bp\n",
      "  Read 2:     6,225,615 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     10,702,515 bp (81.2%)\n",
      "  Read 1:     5,697,023 bp\n",
      "  Read 2:     5,005,492 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 23932 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t45\t0.4\t1\t0 45\n",
      "36\t2\t0.0\t2\t2\n",
      "40\t1\t0.0\t2\t1\n",
      "41\t5\t0.0\t2\t3 2\n",
      "42\t1\t0.0\t2\t1\n",
      "43\t3\t0.0\t2\t2 1\n",
      "44\t9\t0.0\t2\t6 3\n",
      "45\t7\t0.0\t2\t6 1\n",
      "46\t8\t0.0\t2\t8\n",
      "47\t17\t0.0\t2\t17\n",
      "48\t30\t0.0\t2\t28 2\n",
      "49\t63\t0.0\t2\t56 6 1\n",
      "50\t236\t0.0\t2\t182 26 28\n",
      "51\t1345\t0.0\t2\t923 410 12\n",
      "52\t22097\t0.0\t2\t21462 614 21\n",
      "53\t63\t0.0\t2\t33 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 23874 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t2\t0.0\t3\t2\n",
      "45\t1\t0.0\t3\t1\n",
      "46\t1\t0.0\t3\t1\n",
      "47\t6\t0.0\t3\t5 1\n",
      "48\t11\t0.0\t3\t8 2 1\n",
      "49\t114\t0.0\t3\t69 23 20 2\n",
      "50\t1134\t0.0\t3\t703 394 36 1\n",
      "51\t22385\t0.0\t3\t20170 2085 123 7\n",
      "52\t218\t0.0\t3\t176 40 2\n",
      "53\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW025_F11_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW025_F11_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW025_F11_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW025_F11_R03_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW025_F11_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW025_F11_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW025_F11_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW025_F11_R03_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:02        94,907 reads @  26.3 µs/read;   2.28 M reads/minute\n",
      "Finished in 2.498 s (26.324 µs/read; 2.28 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             94,907\n",
      "  Read 1 with adapter:                  94,907 (100.0%)\n",
      "  Read 2 with adapter:                  94,445 (99.5%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                  30 (0.0%)\n",
      "Pairs written (passing filters):        94,416 (99.5%)\n",
      "\n",
      "Total basepairs processed:    51,690,999 bp\n",
      "  Read 1:    27,621,093 bp\n",
      "  Read 2:    24,069,906 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     41,780,135 bp (80.8%)\n",
      "  Read 1:    22,650,596 bp\n",
      "  Read 2:    19,129,539 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 94907 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t428\t1.4\t1\t0 428\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t2\t0.0\t2\t2\n",
      "39\t4\t0.0\t2\t4\n",
      "40\t6\t0.0\t2\t5 1\n",
      "41\t5\t0.0\t2\t5\n",
      "42\t9\t0.0\t2\t9\n",
      "43\t14\t0.0\t2\t11 3\n",
      "44\t11\t0.0\t2\t9 2\n",
      "45\t25\t0.0\t2\t18 6 1\n",
      "46\t38\t0.0\t2\t35 2 1\n",
      "47\t49\t0.0\t2\t45 4\n",
      "48\t70\t0.0\t2\t63 5 2\n",
      "49\t174\t0.0\t2\t153 16 5\n",
      "50\t795\t0.0\t2\t595 94 106\n",
      "51\t5340\t0.0\t2\t3494 1776 70\n",
      "52\t87638\t0.0\t2\t84056 3483 99\n",
      "53\t296\t0.0\t2\t171 120 5\n",
      "54\t1\t0.0\t2\t0 1\n",
      "112\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 94445 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t3\t1\n",
      "39\t1\t0.0\t3\t1\n",
      "41\t3\t0.0\t3\t3\n",
      "42\t1\t0.0\t3\t1\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t1\t0.0\t3\t1\n",
      "46\t3\t0.0\t3\t3\n",
      "47\t9\t0.0\t3\t6 1 0 2\n",
      "48\t15\t0.0\t3\t11 2 2\n",
      "49\t59\t0.0\t3\t39 7 3 10\n",
      "50\t423\t0.0\t3\t267 78 69 9\n",
      "51\t4554\t0.0\t3\t2743 1645 161 5\n",
      "52\t88400\t0.0\t3\t78628 9136 594 42\n",
      "53\t956\t0.0\t3\t783 163 9 1\n",
      "54\t17\t0.0\t3\t9 8\n",
      "57\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW031_F11_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW031_F11_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW031_F11_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW031_F11_R04_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW031_F11_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW031_F11_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW031_F11_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW031_F11_R04_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        22,744 reads @  26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 0.596 s (26.207 µs/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,744\n",
      "  Read 1 with adapter:                  22,744 (100.0%)\n",
      "  Read 2 with adapter:                  22,699 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        22,694 (99.8%)\n",
      "\n",
      "Total basepairs processed:    12,491,108 bp\n",
      "  Read 1:     6,655,385 bp\n",
      "  Read 2:     5,835,723 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     10,189,930 bp (81.6%)\n",
      "  Read 1:     5,469,879 bp\n",
      "  Read 2:     4,720,051 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22744 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t49\t0.3\t1\t0 49\n",
      "39\t2\t0.0\t2\t2\n",
      "40\t2\t0.0\t2\t2\n",
      "41\t4\t0.0\t2\t4\n",
      "42\t2\t0.0\t2\t2\n",
      "43\t5\t0.0\t2\t3 1 1\n",
      "44\t3\t0.0\t2\t2 1\n",
      "45\t8\t0.0\t2\t4 3 1\n",
      "46\t4\t0.0\t2\t4\n",
      "47\t17\t0.0\t2\t17\n",
      "48\t31\t0.0\t2\t29 2\n",
      "49\t43\t0.0\t2\t40 2 1\n",
      "50\t215\t0.0\t2\t154 31 30\n",
      "51\t1269\t0.0\t2\t862 394 13\n",
      "52\t21035\t0.0\t2\t20434 591 10\n",
      "53\t55\t0.0\t2\t37 17 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22699 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t3\t1\n",
      "43\t1\t0.0\t3\t0 1\n",
      "44\t2\t0.0\t3\t2\n",
      "45\t3\t0.0\t3\t2 0 1\n",
      "46\t20\t0.0\t3\t15 2 2 1\n",
      "47\t78\t0.0\t3\t44 14 19 1\n",
      "48\t1097\t0.0\t3\t682 381 33 1\n",
      "49\t21295\t0.0\t3\t19166 1989 136 4\n",
      "50\t199\t0.0\t3\t156 38 5\n",
      "51\t2\t0.0\t3\t2\n",
      "52\t1\t0.0\t3\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW035_F11_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW035_F11_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW035_F11_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW035_F11_R05_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW035_F11_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/GW035_F11_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW035_F11_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/GW035_F11_R05_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        26,327 reads @  25.4 µs/read;   2.36 M reads/minute\n",
      "Finished in 0.671 s (25.477 µs/read; 2.36 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             26,327\n",
      "  Read 1 with adapter:                  26,327 (100.0%)\n",
      "  Read 2 with adapter:                  26,266 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   9 (0.0%)\n",
      "Pairs written (passing filters):        26,257 (99.7%)\n",
      "\n",
      "Total basepairs processed:    14,323,835 bp\n",
      "  Read 1:     7,697,313 bp\n",
      "  Read 2:     6,626,522 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     11,634,976 bp (81.2%)\n",
      "  Read 1:     6,325,659 bp\n",
      "  Read 2:     5,309,317 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 26327 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t67\t0.4\t1\t0 67\n",
      "36\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t1\n",
      "39\t3\t0.0\t2\t3\n",
      "40\t3\t0.0\t2\t3\n",
      "41\t3\t0.0\t2\t3\n",
      "42\t3\t0.0\t2\t3\n",
      "43\t7\t0.0\t2\t6 1\n",
      "44\t10\t0.0\t2\t8 1 1\n",
      "45\t6\t0.0\t2\t5 1\n",
      "46\t7\t0.0\t2\t7\n",
      "47\t19\t0.0\t2\t18 1\n",
      "48\t25\t0.0\t2\t25\n",
      "49\t63\t0.0\t2\t57 6\n",
      "50\t230\t0.0\t2\t182 23 25\n",
      "51\t1498\t0.0\t2\t1010 478 10\n",
      "52\t24299\t0.0\t2\t23532 751 16\n",
      "53\t82\t0.0\t2\t52 30\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 26266 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t3\t1\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t3\t0.0\t3\t3\n",
      "45\t5\t0.0\t3\t4 0 1\n",
      "46\t4\t0.0\t3\t2 2\n",
      "47\t20\t0.0\t3\t11 2 4 3\n",
      "48\t96\t0.0\t3\t64 15 16 1\n",
      "49\t1252\t0.0\t3\t765 448 39\n",
      "50\t24610\t0.0\t3\t21983 2476 136 15\n",
      "51\t270\t0.0\t3\t230 35 5\n",
      "52\t3\t0.0\t3\t2 1\n",
      "53\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH001_F11_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH001_F11_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH001_F11_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH001_F11_R06_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH001_F11_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH001_F11_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH001_F11_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH001_F11_R06_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        22,835 reads @  27.2 µs/read;   2.21 M reads/minute\n",
      "Finished in 0.622 s (27.245 µs/read; 2.20 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,835\n",
      "  Read 1 with adapter:                  22,835 (100.0%)\n",
      "  Read 2 with adapter:                  22,783 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        22,778 (99.8%)\n",
      "\n",
      "Total basepairs processed:    12,504,082 bp\n",
      "  Read 1:     6,667,988 bp\n",
      "  Read 2:     5,836,094 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     10,191,294 bp (81.5%)\n",
      "  Read 1:     5,476,309 bp\n",
      "  Read 2:     4,714,985 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22835 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "8\t44\t0.3\t1\t0 44\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t1\t0.0\t2\t0 1\n",
      "39\t2\t0.0\t2\t2\n",
      "41\t4\t0.0\t2\t4\n",
      "42\t4\t0.0\t2\t4\n",
      "43\t1\t0.0\t2\t1\n",
      "44\t3\t0.0\t2\t3\n",
      "45\t9\t0.0\t2\t5 4\n",
      "46\t9\t0.0\t2\t7 2\n",
      "47\t14\t0.0\t2\t14\n",
      "48\t28\t0.0\t2\t26 1 1\n",
      "49\t74\t0.0\t2\t66 7 1\n",
      "50\t186\t0.0\t2\t149 15 22\n",
      "51\t1271\t0.0\t2\t895 368 8\n",
      "52\t21104\t0.0\t2\t20522 571 11\n",
      "53\t80\t0.0\t2\t46 32 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22783 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t3\t1\n",
      "42\t1\t0.0\t3\t1\n",
      "43\t3\t0.0\t3\t2 1\n",
      "44\t2\t0.0\t3\t2\n",
      "45\t5\t0.0\t3\t4 1\n",
      "46\t16\t0.0\t3\t10 5 0 1\n",
      "47\t78\t0.0\t3\t47 16 12 3\n",
      "48\t932\t0.0\t3\t517 362 53\n",
      "49\t21595\t0.0\t3\t17861 3493 223 18\n",
      "50\t148\t0.0\t3\t118 25 4 1\n",
      "51\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH004_F06_R09_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH004_F06_R09_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH004_F06_R09_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH004_F06_R09_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH004_F06_R09_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH004_F06_R09_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH004_F06_R09_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH004_F06_R09_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        16,549 reads @  26.1 µs/read;   2.30 M reads/minute\n",
      "Finished in 0.434 s (26.211 µs/read; 2.29 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             16,549\n",
      "  Read 1 with adapter:                  16,513 (99.8%)\n",
      "  Read 2 with adapter:                  16,505 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   2 (0.0%)\n",
      "Pairs written (passing filters):        16,473 (99.5%)\n",
      "\n",
      "Total basepairs processed:     9,041,190 bp\n",
      "  Read 1:     4,807,847 bp\n",
      "  Read 2:     4,233,343 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      7,402,547 bp (81.9%)\n",
      "  Read 1:     3,985,876 bp\n",
      "  Read 2:     3,416,671 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 16513 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "38\t1\t0.0\t2\t1\n",
      "42\t2\t0.0\t2\t2\n",
      "43\t1\t0.0\t2\t0 1\n",
      "44\t1\t0.0\t2\t1\n",
      "45\t3\t0.0\t2\t3\n",
      "46\t21\t0.0\t2\t16 2 3\n",
      "47\t140\t0.0\t2\t101 20 19\n",
      "48\t1125\t0.0\t2\t803 311 11\n",
      "49\t15055\t0.0\t2\t14443 597 15\n",
      "50\t164\t0.0\t2\t132 28 4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 16505 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t1\t0.0\t3\t1\n",
      "45\t2\t0.0\t3\t2\n",
      "46\t12\t0.0\t3\t8 2 0 2\n",
      "47\t75\t0.0\t3\t50 12 13\n",
      "48\t811\t0.0\t3\t505 281 21 4\n",
      "49\t15442\t0.0\t3\t13745 1594 99 4\n",
      "50\t161\t0.0\t3\t136 22 3\n",
      "51\t1\t0.0\t3\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH007_F06_R10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH007_F06_R10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH007_F06_R10_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH007_F06_R10_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH007_F06_R10_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH007_F06_R10_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH007_F06_R10_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH007_F06_R10_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         9,111 reads @  27.2 µs/read;   2.21 M reads/minute\n",
      "Finished in 0.250 s (27.433 µs/read; 2.19 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              9,111\n",
      "  Read 1 with adapter:                   9,094 (99.8%)\n",
      "  Read 2 with adapter:                   9,091 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   1 (0.0%)\n",
      "Pairs written (passing filters):         9,074 (99.6%)\n",
      "\n",
      "Total basepairs processed:     4,951,018 bp\n",
      "  Read 1:     2,637,624 bp\n",
      "  Read 2:     2,313,394 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      4,024,863 bp (81.3%)\n",
      "  Read 1:     2,187,292 bp\n",
      "  Read 2:     1,837,571 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 9094 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "40\t1\t0.0\t2\t1\n",
      "44\t2\t0.0\t2\t2\n",
      "46\t8\t0.0\t2\t8\n",
      "47\t81\t0.0\t2\t57 12 12\n",
      "48\t646\t0.0\t2\t453 188 5\n",
      "49\t8267\t0.0\t2\t7955 305 7\n",
      "50\t85\t0.0\t2\t75 10\n",
      "51\t4\t0.0\t2\t4\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 9091 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "45\t1\t0.0\t3\t0 0 1\n",
      "47\t3\t0.0\t3\t3\n",
      "48\t3\t0.0\t3\t2 1\n",
      "49\t5\t0.0\t3\t2 1 1 1\n",
      "50\t50\t0.0\t3\t36 9 4 1\n",
      "51\t488\t0.0\t3\t294 173 16 5\n",
      "52\t8413\t0.0\t3\t6760 1521 123 9\n",
      "53\t126\t0.0\t3\t97 27 2\n",
      "54\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH009_F06_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH009_F06_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH009_F06_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH009_F06_R11_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH009_F06_R11_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH009_F06_R11_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH009_F06_R11_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH009_F06_R11_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        17,711 reads @  26.4 µs/read;   2.27 M reads/minute\n",
      "Finished in 0.469 s (26.495 µs/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             17,711\n",
      "  Read 1 with adapter:                  17,676 (99.8%)\n",
      "  Read 2 with adapter:                  17,684 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):        17,645 (99.6%)\n",
      "\n",
      "Total basepairs processed:     9,762,160 bp\n",
      "  Read 1:     5,170,873 bp\n",
      "  Read 2:     4,591,287 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,010,577 bp (82.1%)\n",
      "  Read 1:     4,293,731 bp\n",
      "  Read 2:     3,716,846 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 17676 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "41\t1\t0.0\t2\t1\n",
      "44\t1\t0.0\t2\t1\n",
      "45\t9\t0.0\t2\t6 2 1\n",
      "46\t19\t0.0\t2\t15 2 2\n",
      "47\t149\t0.0\t2\t116 20 13\n",
      "48\t1242\t0.0\t2\t880 349 13\n",
      "49\t16113\t0.0\t2\t15488 609 16\n",
      "50\t141\t0.0\t2\t120 19 2\n",
      "51\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 17684 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t5\t0.0\t3\t5\n",
      "44\t1\t0.0\t3\t1\n",
      "45\t3\t0.0\t3\t3\n",
      "46\t9\t0.0\t3\t6 0 0 3\n",
      "47\t82\t0.0\t3\t53 14 12 3\n",
      "48\t856\t0.0\t3\t577 257 19 3\n",
      "49\t16529\t0.0\t3\t14899 1538 88 4\n",
      "50\t195\t0.0\t3\t160 30 5\n",
      "51\t4\t0.0\t3\t3 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH014_F06_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH014_F06_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH014_F06_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH014_F06_R12_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH014_F06_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/HH014_F06_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH014_F06_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/HH014_F06_R12_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        11,284 reads @  26.9 µs/read;   2.23 M reads/minute\n",
      "Finished in 0.305 s (27.033 µs/read; 2.22 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             11,284\n",
      "  Read 1 with adapter:                  11,260 (99.8%)\n",
      "  Read 2 with adapter:                  11,257 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):        11,230 (99.5%)\n",
      "\n",
      "Total basepairs processed:     6,211,722 bp\n",
      "  Read 1:     3,299,573 bp\n",
      "  Read 2:     2,912,149 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      5,071,926 bp (81.7%)\n",
      "  Read 1:     2,738,964 bp\n",
      "  Read 2:     2,332,962 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 11260 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "41\t1\t0.0\t2\t1\n",
      "45\t2\t0.0\t2\t0 2\n",
      "46\t15\t0.0\t2\t14 1\n",
      "47\t88\t0.0\t2\t66 12 10\n",
      "48\t766\t0.0\t2\t544 218 4\n",
      "49\t10294\t0.0\t2\t9862 423 9\n",
      "50\t94\t0.0\t2\t75 19\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 11257 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "46\t1\t0.0\t3\t0 1\n",
      "47\t3\t0.0\t3\t1 2\n",
      "48\t7\t0.0\t3\t5 1 1\n",
      "49\t45\t0.0\t3\t33 5 5 2\n",
      "50\t484\t0.0\t3\t291 178 13 2\n",
      "51\t10573\t0.0\t3\t9492 1027 49 5\n",
      "52\t142\t0.0\t3\t128 14\n",
      "53\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN025_F04_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN025_F04_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN025_F04_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN025_F04_R01_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN025_F04_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN025_F04_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN025_F04_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN025_F04_R01_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        14,988 reads @  26.8 µs/read;   2.24 M reads/minute\n",
      "Finished in 0.403 s (26.916 µs/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,988\n",
      "  Read 1 with adapter:                  14,962 (99.8%)\n",
      "  Read 2 with adapter:                  14,945 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   6 (0.0%)\n",
      "Pairs written (passing filters):        14,916 (99.5%)\n",
      "\n",
      "Total basepairs processed:     8,205,542 bp\n",
      "  Read 1:     4,372,348 bp\n",
      "  Read 2:     3,833,194 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      6,705,942 bp (81.7%)\n",
      "  Read 1:     3,627,187 bp\n",
      "  Read 2:     3,078,755 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14962 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "38\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t1\n",
      "44\t2\t0.0\t2\t1 1\n",
      "45\t2\t0.0\t2\t2\n",
      "46\t6\t0.0\t2\t5 0 1\n",
      "47\t102\t0.0\t2\t76 14 12\n",
      "48\t852\t0.0\t2\t548 301 3\n",
      "49\t13796\t0.0\t2\t13392 397 7\n",
      "50\t198\t0.0\t2\t177 21\n",
      "51\t2\t0.0\t2\t2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14945 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t1\t0.0\t3\t0 1\n",
      "45\t4\t0.0\t3\t4\n",
      "46\t2\t0.0\t3\t1 1\n",
      "47\t10\t0.0\t3\t7 1 2\n",
      "48\t47\t0.0\t3\t32 9 6\n",
      "49\t574\t0.0\t3\t336 228 10\n",
      "50\t14156\t0.0\t3\t13003 1095 55 3\n",
      "51\t145\t0.0\t3\t111 32 2\n",
      "52\t5\t0.0\t3\t3 2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN030_F07_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN030_F07_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN030_F07_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN030_F07_R01_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN030_F07_R01_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN030_F07_R01_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN030_F07_R01_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN030_F07_R01_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        18,690 reads @  26.4 µs/read;   2.27 M reads/minute\n",
      "Finished in 0.496 s (26.523 µs/read; 2.26 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             18,690\n",
      "  Read 1 with adapter:                  18,650 (99.8%)\n",
      "  Read 2 with adapter:                  18,654 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   7 (0.0%)\n",
      "Pairs written (passing filters):        18,611 (99.6%)\n",
      "\n",
      "Total basepairs processed:    10,197,000 bp\n",
      "  Read 1:     5,426,719 bp\n",
      "  Read 2:     4,770,281 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,266,753 bp (81.1%)\n",
      "  Read 1:     4,437,126 bp\n",
      "  Read 2:     3,829,627 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 18650 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "41\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t1\n",
      "47\t3\t0.0\t2\t2 0 1\n",
      "48\t3\t0.0\t2\t3\n",
      "49\t8\t0.0\t2\t7 1\n",
      "50\t67\t0.0\t2\t47 4 16\n",
      "51\t456\t0.0\t2\t239 203 14\n",
      "52\t10240\t0.0\t2\t9844 386 10\n",
      "53\t7834\t0.0\t2\t7649 183 2\n",
      "54\t34\t0.0\t2\t23 11\n",
      "55\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 18654 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t3\t1\n",
      "41\t1\t0.0\t3\t0 1\n",
      "42\t1\t0.0\t3\t1\n",
      "44\t2\t0.0\t3\t1 1\n",
      "45\t3\t0.0\t3\t2 0 0 1\n",
      "46\t4\t0.0\t3\t4\n",
      "47\t8\t0.0\t3\t7 1\n",
      "48\t45\t0.0\t3\t27 6 10 2\n",
      "49\t731\t0.0\t3\t433 274 22 2\n",
      "50\t17659\t0.0\t3\t16120 1474 60 5\n",
      "51\t198\t0.0\t3\t158 39 1\n",
      "52\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN035_F04_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN035_F04_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN035_F04_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN035_F04_R02_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN035_F04_R02_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN035_F04_R02_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN035_F04_R02_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN035_F04_R02_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        14,043 reads @  26.7 µs/read;   2.25 M reads/minute\n",
      "Finished in 0.376 s (26.810 µs/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             14,043\n",
      "  Read 1 with adapter:                  14,015 (99.8%)\n",
      "  Read 2 with adapter:                  14,007 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   4 (0.0%)\n",
      "Pairs written (passing filters):        13,979 (99.5%)\n",
      "\n",
      "Total basepairs processed:     7,666,447 bp\n",
      "  Read 1:     4,060,625 bp\n",
      "  Read 2:     3,605,822 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      6,249,465 bp (81.5%)\n",
      "  Read 1:     3,364,341 bp\n",
      "  Read 2:     2,885,124 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 14015 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "43\t3\t0.0\t2\t2 1\n",
      "45\t3\t0.0\t2\t2 0 1\n",
      "46\t12\t0.0\t2\t8 1 3\n",
      "47\t97\t0.0\t2\t67 15 15\n",
      "48\t794\t0.0\t2\t521 264 9\n",
      "49\t12945\t0.0\t2\t12412 520 13\n",
      "50\t158\t0.0\t2\t143 13 2\n",
      "51\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 14007 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "44\t1\t0.0\t3\t1\n",
      "45\t2\t0.0\t3\t2\n",
      "46\t2\t0.0\t3\t2\n",
      "48\t15\t0.0\t3\t10 3 0 2\n",
      "49\t68\t0.0\t3\t42 14 12\n",
      "50\t648\t0.0\t3\t382 245 20 1\n",
      "51\t13137\t0.0\t3\t11727 1324 78 8\n",
      "52\t132\t0.0\t3\t104 25 2 1\n",
      "53\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN038_F04_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN038_F04_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN038_F04_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN038_F04_R03_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN038_F04_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN038_F04_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN038_F04_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN038_F04_R03_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        22,519 reads @  26.7 µs/read;   2.25 M reads/minute\n",
      "Finished in 0.603 s (26.762 µs/read; 2.24 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             22,519\n",
      "  Read 1 with adapter:                  22,470 (99.8%)\n",
      "  Read 2 with adapter:                  22,475 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   8 (0.0%)\n",
      "Pairs written (passing filters):        22,422 (99.6%)\n",
      "\n",
      "Total basepairs processed:    12,341,776 bp\n",
      "  Read 1:     6,525,126 bp\n",
      "  Read 2:     5,816,650 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     10,046,942 bp (81.4%)\n",
      "  Read 1:     5,408,945 bp\n",
      "  Read 2:     4,637,997 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 22470 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t2\t0 1\n",
      "39\t1\t0.0\t2\t1\n",
      "41\t4\t0.0\t2\t4\n",
      "44\t1\t0.0\t2\t1\n",
      "45\t4\t0.0\t2\t2 2\n",
      "46\t14\t0.0\t2\t12 1 1\n",
      "47\t137\t0.0\t2\t91 25 21\n",
      "48\t1288\t0.0\t2\t861 416 11\n",
      "49\t20692\t0.0\t2\t19903 773 16\n",
      "50\t321\t0.0\t2\t283 38\n",
      "51\t7\t0.0\t2\t6 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 22475 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t3\t1\n",
      "38\t1\t0.0\t3\t1\n",
      "44\t1\t0.0\t3\t0 1\n",
      "45\t1\t0.0\t3\t1\n",
      "47\t2\t0.0\t3\t1 1\n",
      "48\t4\t0.0\t3\t2 0 0 2\n",
      "49\t24\t0.0\t3\t14 2 3 5\n",
      "50\t120\t0.0\t3\t75 24 19 2\n",
      "51\t1147\t0.0\t3\t744 370 31 2\n",
      "52\t20917\t0.0\t3\t18800 1981 122 14\n",
      "53\t255\t0.0\t3\t209 43 3\n",
      "54\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN041_F04_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN041_F04_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN041_F04_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN041_F04_R04_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN041_F04_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/MN041_F04_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN041_F04_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/MN041_F04_R04_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        19,063 reads @  26.8 µs/read;   2.24 M reads/minute\n",
      "Finished in 0.513 s (26.920 µs/read; 2.23 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             19,063\n",
      "  Read 1 with adapter:                  19,024 (99.8%)\n",
      "  Read 2 with adapter:                  19,020 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        18,985 (99.6%)\n",
      "\n",
      "Total basepairs processed:    10,336,931 bp\n",
      "  Read 1:     5,521,196 bp\n",
      "  Read 2:     4,815,735 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):      8,454,111 bp (81.8%)\n",
      "  Read 1:     4,577,911 bp\n",
      "  Read 2:     3,876,200 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 19024 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t2\t1\n",
      "42\t1\t0.0\t2\t1\n",
      "43\t1\t0.0\t2\t1\n",
      "44\t5\t0.0\t2\t5\n",
      "45\t9\t0.0\t2\t9\n",
      "46\t22\t0.0\t2\t19 3\n",
      "47\t110\t0.0\t2\t82 16 12\n",
      "48\t1026\t0.0\t2\t692 330 4\n",
      "49\t17590\t0.0\t2\t16974 609 7\n",
      "50\t256\t0.0\t2\t222 34\n",
      "51\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 19020 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t3\t1\n",
      "40\t1\t0.0\t3\t1\n",
      "44\t2\t0.0\t3\t2\n",
      "45\t4\t0.0\t3\t3 0 1\n",
      "46\t19\t0.0\t3\t12 3 2 2\n",
      "47\t93\t0.0\t3\t65 14 12 2\n",
      "48\t1017\t0.0\t3\t662 322 32 1\n",
      "49\t17741\t0.0\t3\t16043 1591 104 3\n",
      "50\t139\t0.0\t3\t114 25\n",
      "51\t2\t0.0\t3\t2\n",
      "64\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR001_F08_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR001_F08_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR001_F08_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR001_F08_R12_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR001_F08_R12_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR001_F08_R12_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR001_F08_R12_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR001_F08_R12_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00           580 reads @  39.5 µs/read;   1.52 M reads/minute\n",
      "Finished in 0.025 s (42.557 µs/read; 1.41 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                580\n",
      "  Read 1 with adapter:                     570 (98.3%)\n",
      "  Read 2 with adapter:                     570 (98.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   0 (0.0%)\n",
      "Pairs written (passing filters):           568 (97.9%)\n",
      "\n",
      "Total basepairs processed:       308,979 bp\n",
      "  Read 1:       167,065 bp\n",
      "  Read 2:       141,914 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):        249,136 bp (80.6%)\n",
      "  Read 1:       137,298 bp\n",
      "  Read 2:       111,838 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 570 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "46\t1\t0.0\t2\t1\n",
      "47\t10\t0.0\t2\t9 1\n",
      "48\t47\t0.0\t2\t34 12 1\n",
      "49\t508\t0.0\t2\t493 15\n",
      "50\t4\t0.0\t2\t1 3\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 570 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "48\t1\t0.0\t3\t1\n",
      "49\t1\t0.0\t3\t1\n",
      "50\t29\t0.0\t3\t19 10\n",
      "51\t527\t0.0\t3\t468 57 2\n",
      "52\t12\t0.0\t3\t11 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR002_F16_R08_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR002_F16_R08_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR002_F16_R08_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR002_F16_R08_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR002_F16_R08_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR002_F16_R08_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR002_F16_R08_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR002_F16_R08_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00         1,012 reads @  34.9 µs/read;   1.72 M reads/minute\n",
      "Finished in 0.037 s (36.996 µs/read; 1.62 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:              1,012\n",
      "  Read 1 with adapter:                     997 (98.5%)\n",
      "  Read 2 with adapter:                   1,001 (98.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   2 (0.2%)\n",
      "Pairs written (passing filters):           994 (98.2%)\n",
      "\n",
      "Total basepairs processed:       526,129 bp\n",
      "  Read 1:       282,244 bp\n",
      "  Read 2:       243,885 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):        422,417 bp (80.3%)\n",
      "  Read 1:       231,194 bp\n",
      "  Read 2:       191,223 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 997 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t2\t0.0\t2\t2\n",
      "45\t1\t0.0\t2\t1\n",
      "47\t11\t0.0\t2\t8 1 2\n",
      "48\t54\t0.0\t2\t40 13 1\n",
      "49\t924\t0.0\t2\t890 32 2\n",
      "50\t5\t0.0\t2\t3 2\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 1001 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "49\t3\t0.0\t3\t3\n",
      "50\t35\t0.0\t3\t18 14 2 1\n",
      "51\t954\t0.0\t3\t837 105 11 1\n",
      "52\t9\t0.0\t3\t8 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR003_F01_R16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR003_F01_R16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR003_F01_R16_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR003_F01_R16_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR003_F01_R16_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/PCR003_F01_R16_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR003_F01_R16_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/PCR003_F01_R16_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00           831 reads @  36.0 µs/read;   1.67 M reads/minute\n",
      "Finished in 0.032 s (38.222 µs/read; 1.57 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:                831\n",
      "  Read 1 with adapter:                     818 (98.4%)\n",
      "  Read 2 with adapter:                     800 (96.3%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   1 (0.1%)\n",
      "Pairs written (passing filters):           797 (95.9%)\n",
      "\n",
      "Total basepairs processed:       433,299 bp\n",
      "  Read 1:       239,694 bp\n",
      "  Read 2:       193,605 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):        344,445 bp (79.5%)\n",
      "  Read 1:       194,296 bp\n",
      "  Read 2:       150,149 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 818 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t2\t1\n",
      "47\t3\t0.0\t2\t2 0 1\n",
      "48\t4\t0.0\t2\t4\n",
      "49\t37\t0.0\t2\t27 9 1\n",
      "50\t772\t0.0\t2\t735 36 1\n",
      "51\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 800 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "47\t1\t0.0\t3\t1\n",
      "48\t3\t0.0\t3\t1 0 2\n",
      "49\t24\t0.0\t3\t4 18 2\n",
      "50\t771\t0.0\t3\t672 90 9\n",
      "51\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY001_F01_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY001_F01_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY001_F01_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY001_F01_R03_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY001_F01_R03_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY001_F01_R03_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY001_F01_R03_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY001_F01_R03_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        32,644 reads @  26.0 µs/read;   2.30 M reads/minute\n",
      "Finished in 0.852 s (26.101 µs/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             32,644\n",
      "  Read 1 with adapter:                  32,575 (99.8%)\n",
      "  Read 2 with adapter:                  32,553 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   7 (0.0%)\n",
      "Pairs written (passing filters):        32,487 (99.5%)\n",
      "\n",
      "Total basepairs processed:    17,945,489 bp\n",
      "  Read 1:     9,615,759 bp\n",
      "  Read 2:     8,329,730 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     14,586,010 bp (81.3%)\n",
      "  Read 1:     7,964,060 bp\n",
      "  Read 2:     6,621,950 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 32575 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t2\t1\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t3\t0.0\t2\t3\n",
      "39\t2\t0.0\t2\t2\n",
      "40\t4\t0.0\t2\t4\n",
      "41\t6\t0.0\t2\t6\n",
      "42\t2\t0.0\t2\t2\n",
      "43\t4\t0.0\t2\t4\n",
      "44\t9\t0.0\t2\t7 2\n",
      "45\t12\t0.0\t2\t8 4\n",
      "46\t23\t0.0\t2\t19 4\n",
      "47\t59\t0.0\t2\t53 4 2\n",
      "48\t299\t0.0\t2\t236 31 32\n",
      "49\t1806\t0.0\t2\t1249 539 18\n",
      "50\t30238\t0.0\t2\t29153 1055 30\n",
      "51\t104\t0.0\t2\t73 30 1\n",
      "52\t2\t0.0\t2\t1 0 1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 32553 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t3\t1\n",
      "42\t1\t0.0\t3\t1\n",
      "45\t2\t0.0\t3\t2\n",
      "46\t3\t0.0\t3\t3\n",
      "47\t2\t0.0\t3\t2\n",
      "48\t4\t0.0\t3\t4\n",
      "49\t30\t0.0\t3\t24 5 0 1\n",
      "50\t169\t0.0\t3\t109 32 23 5\n",
      "51\t1706\t0.0\t3\t1093 568 42 3\n",
      "52\t30290\t0.0\t3\t27247 2870 165 8\n",
      "53\t343\t0.0\t3\t280 59 4\n",
      "54\t2\t0.0\t3\t2\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY005_F01_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY005_F01_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY005_F01_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY005_F01_R04_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY005_F01_R04_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY005_F01_R04_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY005_F01_R04_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY005_F01_R04_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        28,465 reads @  26.0 µs/read;   2.31 M reads/minute\n",
      "Finished in 0.741 s (26.033 µs/read; 2.30 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,465\n",
      "  Read 1 with adapter:                  28,410 (99.8%)\n",
      "  Read 2 with adapter:                  28,399 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                  14 (0.0%)\n",
      "Pairs written (passing filters):        28,336 (99.5%)\n",
      "\n",
      "Total basepairs processed:    15,538,941 bp\n",
      "  Read 1:     8,331,321 bp\n",
      "  Read 2:     7,207,620 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,695,393 bp (81.7%)\n",
      "  Read 1:     6,891,097 bp\n",
      "  Read 2:     5,804,296 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28410 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "36\t1\t0.0\t2\t1\n",
      "37\t2\t0.0\t2\t2\n",
      "38\t3\t0.0\t2\t3\n",
      "40\t2\t0.0\t2\t2\n",
      "42\t5\t0.0\t2\t5\n",
      "43\t4\t0.0\t2\t2 2\n",
      "44\t10\t0.0\t2\t7 3\n",
      "45\t12\t0.0\t2\t6 5 1\n",
      "46\t20\t0.0\t2\t19 1\n",
      "47\t57\t0.0\t2\t56 1\n",
      "48\t237\t0.0\t2\t178 30 29\n",
      "49\t1683\t0.0\t2\t1166 500 17\n",
      "50\t26289\t0.0\t2\t25399 864 26\n",
      "51\t84\t0.0\t2\t65 18 1\n",
      "52\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 28399 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t3\t1\n",
      "41\t1\t0.0\t3\t0 0 1\n",
      "42\t2\t0.0\t3\t2\n",
      "43\t1\t0.0\t3\t1\n",
      "44\t2\t0.0\t3\t2\n",
      "45\t7\t0.0\t3\t5 1 1\n",
      "46\t19\t0.0\t3\t7 4 1 7\n",
      "47\t150\t0.0\t3\t87 33 28 2\n",
      "48\t1411\t0.0\t3\t900 465 44 2\n",
      "49\t26578\t0.0\t3\t23888 2555 132 3\n",
      "50\t226\t0.0\t3\t178 47 1\n",
      "51\t1\t0.0\t3\t0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY009_F01_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY009_F01_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY009_F01_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY009_F01_R05_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY009_F01_R05_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY009_F01_R05_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY009_F01_R05_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY009_F01_R05_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        28,965 reads @  26.6 µs/read;   2.26 M reads/minute\n",
      "Finished in 0.771 s (26.624 µs/read; 2.25 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             28,965\n",
      "  Read 1 with adapter:                  28,886 (99.7%)\n",
      "  Read 2 with adapter:                  28,918 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   3 (0.0%)\n",
      "Pairs written (passing filters):        28,842 (99.6%)\n",
      "\n",
      "Total basepairs processed:    15,847,530 bp\n",
      "  Read 1:     8,499,676 bp\n",
      "  Read 2:     7,347,854 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     12,919,480 bp (81.5%)\n",
      "  Read 1:     7,033,560 bp\n",
      "  Read 2:     5,885,920 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 28886 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t2\t1\n",
      "39\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t1\n",
      "41\t2\t0.0\t2\t2\n",
      "42\t1\t0.0\t2\t1\n",
      "43\t7\t0.0\t2\t6 1\n",
      "44\t8\t0.0\t2\t6 2\n",
      "45\t13\t0.0\t2\t7 6\n",
      "46\t26\t0.0\t2\t21 4 1\n",
      "47\t49\t0.0\t2\t45 3 1\n",
      "48\t236\t0.0\t2\t191 23 22\n",
      "49\t1579\t0.0\t2\t1107 460 12\n",
      "50\t26876\t0.0\t2\t25943 918 15\n",
      "51\t86\t0.0\t2\t55 31\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 28918 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "39\t1\t0.0\t3\t1\n",
      "40\t2\t0.0\t3\t1 1\n",
      "41\t1\t0.0\t3\t1\n",
      "42\t1\t0.0\t3\t1\n",
      "43\t5\t0.0\t3\t4 1\n",
      "44\t3\t0.0\t3\t3\n",
      "45\t1\t0.0\t3\t1\n",
      "46\t3\t0.0\t3\t2 0 0 1\n",
      "47\t32\t0.0\t3\t17 9 4 2\n",
      "48\t97\t0.0\t3\t55 16 21 5\n",
      "49\t1290\t0.0\t3\t773 476 39 2\n",
      "50\t27183\t0.0\t3\t24241 2739 194 9\n",
      "51\t296\t0.0\t3\t253 39 4\n",
      "52\t3\t0.0\t3\t2 0 1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY014_F01_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY014_F01_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY014_F01_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY014_F01_R06_L001_R2_001.fastq.gz\n",
      "\n",
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY014_F01_R06_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY014_F01_R06_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY014_F01_R06_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY014_F01_R06_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        29,485 reads @  25.9 µs/read;   2.31 M reads/minute\n",
      "Finished in 0.767 s (26.007 µs/read; 2.31 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             29,485\n",
      "  Read 1 with adapter:                  29,430 (99.8%)\n",
      "  Read 2 with adapter:                  29,432 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   6 (0.0%)\n",
      "Pairs written (passing filters):        29,381 (99.6%)\n",
      "\n",
      "Total basepairs processed:    15,992,358 bp\n",
      "  Read 1:     8,597,710 bp\n",
      "  Read 2:     7,394,648 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     13,053,067 bp (81.6%)\n",
      "  Read 1:     7,111,288 bp\n",
      "  Read 2:     5,941,779 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 29430 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t1\t0.0\t2\t1\n",
      "38\t3\t0.0\t2\t3\n",
      "40\t4\t0.0\t2\t4\n",
      "41\t2\t0.0\t2\t2\n",
      "42\t4\t0.0\t2\t4\n",
      "43\t7\t0.0\t2\t6 1\n",
      "44\t7\t0.0\t2\t5 1 1\n",
      "45\t10\t0.0\t2\t9 0 1\n",
      "46\t27\t0.0\t2\t24 2 1\n",
      "47\t52\t0.0\t2\t44 7 1\n",
      "48\t235\t0.0\t2\t189 29 17\n",
      "49\t1657\t0.0\t2\t1181 460 16\n",
      "50\t27321\t0.0\t2\t26437 869 15\n",
      "51\t100\t0.0\t2\t63 37\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 29432 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "35\t1\t0.0\t3\t1\n",
      "39\t1\t0.0\t3\t1\n",
      "40\t1\t0.0\t3\t1\n",
      "41\t1\t0.0\t3\t1\n",
      "42\t2\t0.0\t3\t1 1\n",
      "43\t4\t0.0\t3\t2 1 1\n",
      "44\t6\t0.0\t3\t5 1\n",
      "45\t5\t0.0\t3\t4 0 1\n",
      "46\t20\t0.0\t3\t11 3 2 4\n",
      "47\t108\t0.0\t3\t63 20 24 1\n",
      "48\t1236\t0.0\t3\t679 468 88 1\n",
      "49\t27840\t0.0\t3\t23039 4470 312 19\n",
      "50\t204\t0.0\t3\t153 46 4 1\n",
      "51\t2\t0.0\t3\t2\n",
      "73\t1\t0.0\t3\t1\n",
      "\n",
      "Command: cutadapt --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY018_F01_R08_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY018_F01_R08_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY018_F01_R08_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY018_F01_R08_L001_R2_001.fastq.gz\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is cutadapt 4.9 with Python 3.9.19\n",
      "Command line parameters: --cores 1 --error-rate 0.15 --times 1 --overlap 3 --minimum-length 1 -q 0,0 --quality-base 33 -o /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY018_F01_R08_L001_R1_001.fastq.gz -p /tmp/q2-CasavaOneEightSingleLanePerSampleDirFmt-i1h4zl6y/TY018_F01_R08_L001_R2_001.fastq.gz --front GTGCCAGCMGCCGCGGTAA -G GGACTACHVGGGTWTCTAAT --discard-untrimmed /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY018_F01_R08_L001_R1_001.fastq.gz /tmp/qiime2/root/data/d53d6a71-d43a-4737-a1d9-7be372cd48d2/data/TY018_F01_R08_L001_R2_001.fastq.gz\n",
      "Processing paired-end reads on 1 core ...\n",
      "Done           00:00:00        25,753 reads @  25.4 µs/read;   2.36 M reads/minute\n",
      "Finished in 0.657 s (25.521 µs/read; 2.35 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total read pairs processed:             25,753\n",
      "  Read 1 with adapter:                  25,696 (99.8%)\n",
      "  Read 2 with adapter:                  25,697 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Pairs that were too short:                   5 (0.0%)\n",
      "Pairs written (passing filters):        25,639 (99.6%)\n",
      "\n",
      "Total basepairs processed:    13,981,121 bp\n",
      "  Read 1:     7,511,932 bp\n",
      "  Read 2:     6,469,189 bp\n",
      "Quality-trimmed:                       0 bp (0.0%)\n",
      "  Read 1:             0 bp\n",
      "  Read 2:             0 bp\n",
      "Total written (filtered):     11,354,987 bp (81.2%)\n",
      "  Read 1:     6,208,914 bp\n",
      "  Read 2:     5,146,073 bp\n",
      "\n",
      "=== First read: Adapter 1 ===\n",
      "\n",
      "Sequence: GTGCCAGCMGCCGCGGTAA; Type: regular 5'; Length: 19; Trimmed: 25696 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "37\t2\t0.0\t2\t2\n",
      "38\t3\t0.0\t2\t3\n",
      "39\t1\t0.0\t2\t1\n",
      "40\t1\t0.0\t2\t1\n",
      "41\t4\t0.0\t2\t3 1\n",
      "42\t3\t0.0\t2\t2 1\n",
      "43\t4\t0.0\t2\t4\n",
      "44\t10\t0.0\t2\t5 5\n",
      "45\t9\t0.0\t2\t8 1\n",
      "46\t23\t0.0\t2\t21 2\n",
      "47\t44\t0.0\t2\t40 3 1\n",
      "48\t237\t0.0\t2\t187 27 23\n",
      "49\t1496\t0.0\t2\t1028 461 7\n",
      "50\t23773\t0.0\t2\t23034 718 21\n",
      "51\t85\t0.0\t2\t65 20\n",
      "52\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "=== Second read: Adapter 2 ===\n",
      "\n",
      "Sequence: GGACTACHVGGGTWTCTAAT; Type: regular 5'; Length: 20; Trimmed: 25697 times\n",
      "\n",
      "Minimum overlap: 3\n",
      "No. of allowed errors:\n",
      "1-5 bp: 0; 6-12 bp: 1; 13-19 bp: 2; 20 bp: 3\n",
      "\n",
      "Overview of removed sequences\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "44\t2\t0.0\t3\t1 1\n",
      "45\t2\t0.0\t3\t1 0 1\n",
      "46\t4\t0.0\t3\t4\n",
      "47\t3\t0.0\t3\t3\n",
      "48\t19\t0.0\t3\t9 4 0 6\n",
      "49\t95\t0.0\t3\t52 22 19 2\n",
      "50\t1174\t0.0\t3\t704 435 34 1\n",
      "51\t24171\t0.0\t3\t22006 2044 111 10\n",
      "52\t224\t0.0\t3\t176 47 1\n",
      "53\t2\t0.0\t3\t2\n",
      "54\t1\t0.0\t3\t1\n",
      "\u001b[32mSaved SampleData[PairedEndSequencesWithQuality] to: trimmed_sequences/trimmed_sequences.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime cutadapt trim-paired \\\n",
    "  --i-demultiplexed-sequences demux.qza \\\n",
    "  --p-front-f GTGCCAGCMGCCGCGGTAA \\\n",
    "  --p-front-r GGACTACHVGGGTWTCTAAT \\\n",
    "  --p-discard-untrimmed \\\n",
    "  --p-error-rate 0.15 \\\n",
    "  --output-dir trimmed_sequences \\\n",
    "  --verbose"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "609def45",
   "metadata": {},
   "source": [
    "### Visualizing the demultiplexed sequences quality control graphs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "39b97bcf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: demux.qzv\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime demux summarize \\\n",
    "  --i-data trimmed_sequences/trimmed_sequences.qza \\\n",
    "  --o-visualization demux.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4e2fbf6e",
   "metadata": {},
   "source": [
    "### 4 Denoising sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "2383aa24",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: table.qza\u001b[0m\n",
      "\u001b[32mSaved FeatureData[Sequence] to: rep-seqs.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[DADA2Stats] to: denoising-stats.qza\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime dada2 denoise-paired \\\n",
    "  --i-demultiplexed-seqs trimmed_sequences/trimmed_sequences.qza \\\n",
    "  --p-trim-left-f 0 \\\n",
    "  --p-trim-left-r 0 \\\n",
    "  --p-trunc-len-f 150 \\\n",
    "  --p-trunc-len-r 150 \\\n",
    "  --o-table table.qza \\\n",
    "  --o-representative-sequences rep-seqs.qza \\\n",
    "  --o-denoising-stats denoising-stats.qza"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "565c1355",
   "metadata": {},
   "source": [
    "### 5 Generating feature table summary and visualizing representative sequences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "a36fc020",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: table.qzv\u001b[0m\n",
      "\u001b[0m\u001b[32mSaved Visualization to: rep-seqs.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime feature-table summarize \\\n",
    "  --i-table table.qza \\\n",
    "  --o-visualization table.qzv \\\n",
    "  --m-sample-metadata-file echidna_metadata.tsv\n",
    "!qiime feature-table tabulate-seqs \\\n",
    "  --i-data rep-seqs.qza \\\n",
    "  --o-visualization rep-seqs.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0f5e9b96",
   "metadata": {},
   "source": [
    "### 6 Phylogenetic analysis : Generation of rooted and unrooted trees¶"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "f20bcf04",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureData[AlignedSequence] to: aligned-rep-seqs.qza\u001b[0m\r\n",
      "\u001b[32mSaved FeatureData[AlignedSequence] to: masked-aligned-rep-seqs.qza\u001b[0m\r\n",
      "\u001b[32mSaved Phylogeny[Unrooted] to: unrooted-tree.qza\u001b[0m\r\n",
      "\u001b[32mSaved Phylogeny[Rooted] to: rooted-tree.qza\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime phylogeny align-to-tree-mafft-fasttree \\\n",
    "  --i-sequences rep-seqs.qza \\\n",
    "  --o-alignment aligned-rep-seqs.qza \\\n",
    "  --o-masked-alignment masked-aligned-rep-seqs.qza \\\n",
    "  --o-tree unrooted-tree.qza \\\n",
    "  --o-rooted-tree rooted-tree.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "129d2a70",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved FeatureTable[Frequency] to: core-metrics-results/rarefied_table.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/faith_pd_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/observed_features_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/shannon_vector.qza\u001b[0m\n",
      "\u001b[32mSaved SampleData[AlphaDiversity] to: core-metrics-results/evenness_vector.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: core-metrics-results/unweighted_unifrac_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: core-metrics-results/weighted_unifrac_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: core-metrics-results/jaccard_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved DistanceMatrix to: core-metrics-results/bray_curtis_distance_matrix.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: core-metrics-results/unweighted_unifrac_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: core-metrics-results/weighted_unifrac_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: core-metrics-results/jaccard_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved PCoAResults to: core-metrics-results/bray_curtis_pcoa_results.qza\u001b[0m\n",
      "\u001b[32mSaved Visualization to: core-metrics-results/unweighted_unifrac_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: core-metrics-results/weighted_unifrac_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: core-metrics-results/jaccard_emperor.qzv\u001b[0m\n",
      "\u001b[32mSaved Visualization to: core-metrics-results/bray_curtis_emperor.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime diversity core-metrics-phylogenetic \\\n",
    "  --i-phylogeny rooted-tree.qza \\\n",
    "  --i-table table.qza \\\n",
    "  --p-sampling-depth 120  \\\n",
    "  --m-metadata-file echidna_metadata.tsv \\\n",
    "  --output-dir core-metrics-results"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8293e155",
   "metadata": {},
   "source": [
    "### 7 Alpha rarefaction plotting"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "7c8051a2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: alpha-rarefaction.qzv\u001b[0m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime diversity alpha-rarefaction \\\n",
    "  --i-table table.qza \\\n",
    "  --i-phylogeny rooted-tree.qza \\\n",
    "  --p-max-depth 114 \\\n",
    "  --m-metadata-file echidna_metadata.tsv \\\n",
    "  --o-visualization alpha-rarefaction.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bb912400",
   "metadata": {},
   "source": [
    "### 8 Beta rarefaction plotting"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "b37aa505",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: beta-rarefaction.qzv\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime diversity beta-rarefaction \\\n",
    "    --i-table table.qza \\\n",
    "    --i-phylogeny rooted-tree.qza \\\n",
    "    --p-metric weighted_unifrac \\\n",
    "    --p-clustering-method upgma \\\n",
    "    --m-metadata-file echidna_metadata.tsv \\\n",
    "    --p-sampling-depth 114 \\\n",
    "    --o-visualization beta-rarefaction.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3ac03528",
   "metadata": {},
   "source": [
    "### 9 Taxonomic classification"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bad6609a",
   "metadata": {},
   "source": [
    "This step deals with the exploration of taxonomic composition of the samples, and again relate that to sample metadata. The first step in this process is to assign taxonomy to the sequences in our FeatureData[Sequence] QIIME 2 artifact. We’ll do that using a pre-trained Naive Bayes classifier and the q2-feature-classifier plugin. This classifier was trained on the Greengenes 13_8 99% OTUs, where the sequences have been trimmed to only include 250 bases from the region of the 16S that was sequenced in this analysis (the V4 region, bound by the 515F/806R primer pair). We’ll apply this classifier to our sequences, and we can generate a visualization of the resulting mapping from sequence to taxonomy."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "7bdd677a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting scikit-learn==0.24.1\n",
      "  Using cached scikit_learn-0.24.1-cp39-cp39-manylinux2010_x86_64.whl.metadata (9.7 kB)\n",
      "Requirement already satisfied: numpy>=1.13.3 in /root/anaconda3/envs/qiime2-amplicon-2024.5/lib/python3.9/site-packages (from scikit-learn==0.24.1) (1.26.4)\n",
      "Requirement already satisfied: scipy>=0.19.1 in /root/anaconda3/envs/qiime2-amplicon-2024.5/lib/python3.9/site-packages (from scikit-learn==0.24.1) (1.13.0)\n",
      "Requirement already satisfied: joblib>=0.11 in /root/anaconda3/envs/qiime2-amplicon-2024.5/lib/python3.9/site-packages (from scikit-learn==0.24.1) (1.4.2)\n",
      "Requirement already satisfied: threadpoolctl>=2.0.0 in /root/anaconda3/envs/qiime2-amplicon-2024.5/lib/python3.9/site-packages (from scikit-learn==0.24.1) (3.5.0)\n",
      "Using cached scikit_learn-0.24.1-cp39-cp39-manylinux2010_x86_64.whl (23.8 MB)\n",
      "Installing collected packages: scikit-learn\n",
      "  Attempting uninstall: scikit-learn\n",
      "    Found existing installation: scikit-learn 1.4.2\n",
      "    Uninstalling scikit-learn-1.4.2:\n",
      "      Successfully uninstalled scikit-learn-1.4.2\n",
      "Successfully installed scikit-learn-0.24.1\n",
      "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!pip install scikit-learn==0.24.1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "6289b98f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.24.1\r\n"
     ]
    }
   ],
   "source": [
    "!python -c \"import sklearn; print(sklearn.__version__)\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "6a317cdd",
   "metadata": {},
   "outputs": [],
   "source": [
    "!qiime feature-classifier classify-sklearn \\\n",
    "  --i-classifier silva_138_16s_515-806_classifier.qza \\\n",
    "  --i-reads rep-seqs.qza \\\n",
    "  --o-classification taxonomy.qza"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "cc68a2a8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[32mSaved Visualization to: taxonomy.qzv\u001b[0m\r\n",
      "\u001b[0m"
     ]
    }
   ],
   "source": [
    "!qiime metadata tabulate \\\n",
    "  --m-input-file taxonomy.qza \\\n",
    "  --o-visualization taxonomy.qzv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "51aaaef5",
   "metadata": {},
   "source": [
    "### 10 Visualizing taxonomy bar plots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4865aac8",
   "metadata": {},
   "outputs": [],
   "source": [
    "!qiime taxa barplot \\\n",
    "  --i-table table.qza \\\n",
    "  --i-taxonomy taxonomy.qza \\\n",
    "  --m-metadata-file echidna_metadata.tsv \\\n",
    "  --o-visualization taxa-bar-plots.qzv\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
